### ISSUE TITLE
Replace the current paths system

### ISSUE THREAD

**Issue by wvanbreukelen** [wvanbreukelen/FlyerCore#18]
Using functions like app_path(), the developer can access a various choices of path constants.

An issue with this, is that PHP's architecture does not allow to call classes inside a function. Maybe a use-statement will be an option?

```php
<?php

if (!function_exists('app'))
{
	function app($method)
	{
		return App::getInstance()->{$method}();
	}
}


function base_path()
{
	return app('basePath');
}
```

There must be a way that the app function can been implemented like in Laravel 4/5.

**Comment by wvanbreukelen** [comment]
Laravel helper functions: http://laravel.com/docs/5.0/helpers

**Comment by wvanbreukelen** [comment]
Implemented fix in app.php

**Comment by wvanbreukelen** [comment]
Fixed with 27cf1e7
